packages/address-table/contracts/ArbitrumVIP.sol

```diff
@@ -1,8 +1,8 @@
 //SPDX-License-Identifier: Unlicense
-pragma solidity ^0.7.0;
+pragma solidity  >=0.6.11;
 
+import "arbos-precompiles/arbos/builtin/ArbAddressTable.sol";
 import "hardhat/console.sol";
-import "arb-shared-dependencies/contracts/ArbAddressTable.sol";
```

Comment:
We could delete the smart contracts from `arb-shared-dependencies` that aren't used anymore (i think it will prob be all of them)